Formulario de creación y edición de categorías

DiegoSantelices10 opened the pull request:
se realizó el formulario de creación y edición de categorías, reutilizando un solo formulario para las dos opciones.

![Screenshot_5](https://user-images.githubusercontent.com/70447941/199564553-a1ce00c7-476e-471b-903e-8db43d10b24b.png)
![Screenshot_6](https://user-images.githubusercontent.com/70447941/199564594-03f550c7-9d12-4af4-a321-be9acac47ed1.png)

DiegoSantelices10 committed:
Tarea Completada

se realizó el formulario de creacion y edicion de categorias, reutilizando un solo formulario para las dos opciones.

SoraBeat commented:
Falta borrar este comentario xd
![image](https://user-images.githubusercontent.com/30334118/199793725-f9f43cf5-9740-44a9-9bcd-1c9b76e59fc2.png)
Cambiar el mensaje de error al submitear y no haber seleccionado una imagen
![image](https://user-images.githubusercontent.com/30334118/199794171-4d2c9cca-9445-4704-95fe-8e223a645388.png)
Al insertar un video no me salta error de tipo de archivo (png jpg)
![image](https://user-images.githubusercontent.com/30334118/199794638-d0023669-d8b4-481b-9542-877c003488c3.png)

DiegoSantelices10 committed:
Validación de imagen

se establecio como validación de formato .png .jpg

DiegoSantelices10 committed:
update

DiegoSantelices10 committed:
Update Categoria

Se actualizo la validación de la imagen, y se soluciono los valores iniciales de los input

DiegoSantelices10 committed:
Optimización de codigo.

Seguimos optimizando el componente de categorias

milandre commented on src/Components/Categories/CategoriesForm.js:
```diff
@@ -1,32 +1,178 @@
+					}}
+					validationSchema={() =>
+						yup.object().shape({
+							name: yup.string().required(message),
```
Falta validación de longitud minima de 4 carateres

milandre commented on src/Components/Categories/CategoriesForm.js:
```diff
@@ -1,32 +1,178 @@
+	};
+
+	return (
+		<div className="h-screen">
```
Los estilos y estructura de este formulario deben ser consistentes con los otros formularios que han sido creados para la parte del administrador. Sugiero reunirse con @SoraBeat, @YonaBordon, @gabrielpindo y ponerse de acuerdo con esto.

DiegoSantelices10 replied:
Dale Andre queda pendiente esta corrección para cuando definamos una estructura con mis compañeros.

DiegoSantelices10 committed:
Update de formulario

Se realizaron las correcciones requeridas!

DiegoSantelices10 committed:
update

creacion de carpeta Utils para funciones reutilizables

DiegoSantelices10 committed:
update

DiegoSantelices10 committed:
Merge branch 'dev' of https://github.com/alkemyTech/React-SXXI-T4 into feature/ONG4-28/formularioEdiciónCreaciónDeCategorias

DiegoSantelices10 committed:
update

DiegoSantelices10 committed:
update de formulario

se modifico el diseño del formulario que vamos a utilizar para el backoffice,

DiegoSantelices10 commented:
Andre ahi resolvimos el diseño del formulario para todo el backoffice ya realice los cambio en esta rama

DiegoSantelices10 commented:
![Screenshot_15](https://user-images.githubusercontent.com/70447941/201796045-4f3a2812-5672-4b92-83ca-5d3682318313.png)

DiegoSantelices10 committed:
Correccion de imagen

se realizo la validacion para cuando una imagen no es modificada se pueda guardar el resto con exito!

DiegoSantelices10 committed:
Merge branch 'dev' into feature/ONG4-28/formularioEdiciónCreaciónDeCategorias
